wear/src/main/java/io/homeassistant/companion/android/home/views/HomeView.kt

```diff
@@ -222,8 +222,10 @@ fun LoadHomePage(
                 val sensorManager = getSensorManagers().first { sensorManager ->
                     sensorManager.id() == sensorManagerId
                 }
+                mainViewModel.updateAllSensors(sensorManager)
                 SensorManagerUi(
                     allSensors = mainViewModel.sensors.value,
+                    allAvailSensors = mainViewModel.availableSensors,
```

Comment:
Assume I'm first looking at sensor manager A, which loads its data into `allAvailSensors`, and then I go back and want to look at sensor manager B. In that case I think the data for A would still be in here and you could see incorrect sensors while the data for B is being loaded?

Reply:
Just tested and this is not the case here, navigating between each manager still only shows its respective available sensor list :)

Reply:
In testing that is also what I see, but do you agree it could happen? Take for example the 17 seconds you experienced on first load, what if that was sensor manager B?

Reply:
Honestly I am not sure, almost seems like compose is smart enough to see the multiple instances and keep separate values. That is a big guess on my part :)

Reply:
Health services don't seem to work on the emulator so I can confirm that it keeps the old values

https://user-images.githubusercontent.com/8148535/198901045-77b6b65a-58ad-405b-82eb-d25fbabf3081.mp4

Reply:
hmm trying to think of whats the best way to handle this so the view model can continue to provide the data 🤔  I suppose I could clear the data on each update call to reset it?

Reply:
ok this should be fixed in the latest commit now :) appreciate the video and testing!